css-layout-classes/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>CSS-Layout-Classes</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" />
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
```

Comment:
There are four columns here, so they should have a class which matches and has the proper width setting for 1/4 of the row.